doc/CRD-generator.md

```diff
@@ -444,6 +465,7 @@ The CRD generator will add the additional `labels`:
 | `io.fabric8.crd.generator.annotation.SchemaSwap`             | Similar to SchemaFrom, but can be applied at any point in the class hierarchy            |
 | `io.fabric8.crd.generator.annotation.Annotations`            | Additional `annotations` in `metadata`                                                |
 | `io.fabric8.crd.generator.annotation.Labels`                 | Additional `labels` in `metadata`                                                     |
+| `io.fabric8.kubernetes.model.annotation.PrinterColumn`       | Customize columns shown by the `kubectl get` command                                  |
```

Comment:
Thanks a lot for adding this entry!
I actually think that this is a leftover and we should move this to the `generator-annotations` module.
(we can tackle this issue outside the scope of this PR)

cc. @manusa @shawkins @metacosm

Reply:
We should go through something like this:
https://github.com/bachmanity1/kubernetes-client/pull/1

to actually remove the annotation.
Since it was completely undocumented, I would get this change in **before** we release the next 6.8 if possible.

Reply:
Can I handle it in a separate PR?